libs/vectorise/include/vectorise/fixed_point/fixed_point.hpp

```diff
@@ -568,7 +568,7 @@ class FixedPoint : public BaseFixedpointType
 };
 
 template <typename T>
-static inline bool IsMinusZero(T n);
+bool IsMinusZero(T n);
```

Comment:
Why the drop of `static`

Reply:
What's the point of having it there? All it does is confer internal linkage, which is meaningless in a header file.